cmd/rancher_release/README.md

```diff
@@ -4,7 +4,115 @@ rancher_release is a utility that performs a number of tasks related to the Ranc
+
+| **Flag** | **Description**        | **Required** |
+| -------- | ---------------------- | ------------ |
+| tag      | Release tag in GitHub. | TRUE         |
+
+**Examples**
+
+```
+rancher_release list-nonmirrored-rc-images —tag v2.6
+```
+
+### check-rancher-image
+
+Checks if there’s an available Helm Chart and Docker images for amd64, arm and s390x for a given tag.
+
+| **Flag** | **Description**        | **Required** |
+| -------- | ---------------------- | ------------ |
+| tag      | Release tag in GitHub. | TRUE         |
+
+**Examples**
+
+```
+rancher_release check-rancher-image —tag v2.6
+```
+
+### set-kdm-branch-refs
+
+Updates Rancher KDM branch references in:
+
+- `pkg/settings/setting.go`
+- `package/Dockerfile`
+- `Dockerfile.dapper`
+
+| **Flag**           | **Description**                                                                                                                                                                               | **Required** |
+| ------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------ |
+| fork-path          | Path for your fork of rancher/rancher.                                                                                                                                                        | TRUE         |
+| base-branch        | The branch you want to update with the new KDM branch.                                                                                                                                        | TRUE         |
+| current-kdm-branch | Current KDM branch used in the files listed above.                                                                                                                                            | TRUE         |
+| new-kdm-branch     | KDM branch to replace the current.                                                                                                                                                            | TRUE         |
+| create-pr          | if true, will try to create a PR against the `base-branch` in rancher/rancher, may fail if your GitHub token doesn’t have the required permission. Requires a GITHUB_TOKEN env var to be set. | FALSE        |
+| fork-owner         | GitHub Username of the owner of the rancher fork used in `rancher-fork`.                                                                                                                      | FALSE        |
+| dry-run            | Changes will not be pushed to remote and the PR will not be created.                                                                                                                          | FALSE        |
+
+**Examples**
+
+```
+rancher_release set-kdm-branch-refs —fork-path $GOPATH/src/github.com/{YOUR_USERNAME}/rancher \
+    —base-branch release/v2.8 \
```

Comment:
I don't know why I didn't notice this earlier but we need to update the flags to conform to standard UNIX flag standards. ie `-f` `--fork-path`, etc. Single hyphen for a single flag and double hyphen for full words.